C++/C++.sublime-syntax

```diff
@@ -759,6 +762,26 @@ contexts:
               scope: keyword.declaration.c++
             - include: template-common
 
+  requires:
+    - match: \brequires\b
+      scope: keyword.operator.word.c++
```

Comment:
What's the purpose of scoping requires `keyword.operator` hier, while using `storage.modifier` in `constraint-requires`?

Reply:
they both mean different things, one is a keyword to check if a function is valid and the other is a way to begin a constraint expression outside of a concept.